lib/ui/components/common/movie_item.dart

```diff
@@ -89,3 +89,9 @@ class MovieItem extends StatelessWidget {
     );
   }
 }
+
+String getYearOfMovie(Movie movie) {
+  return movie.releaseDate?.year != null
+      ? movie.releaseDate.year.toString()
+      : 'Unknown';
```

Comment:
```suggestion
  return movie.releaseDate?.year?.toString() ?? 'Unknown';
```
anh thử cái này xem được không :v